onecodex/vendored/potion_client/links.py

```diff
@@ -92,6 +92,9 @@ def raise_for_status(self, response):
     def make_request(self, data, params):
         req = self.request_factory(data, params)
         prepared_request = self.owner._client.session.prepare_request(req)
+        settings = self.owner._client.session.merge_environment_settings(
```

Comment:
This needs a comment as to _why_ we're modifying the session and linking to the requests docs. Also, I _think_ we should be calling this right after we instantiate the client in the main `__init__` method (off of `session`)? That's probably better even if 100% of requests flow through `make_request` in `potion_client/links` (which I'm not 100% sure of).

Reply:
Ideally, we'd also have a test showing the `CA_BUNDLE...` gets loaded. We can just mock the environment in pytest.

Reply:
OK I added a request to ensure that we're calling `merge_environment_settings`. It's not great, but better than missing it I suppose.